distro/vertx/src/main/resources/overlay/configs/conf-3scale-immutable-es.json

```diff
@@ -72,7 +72,10 @@
 
     "plugin-registry": {
       "class": "io.apiman.gateway.platforms.vertx3.engine.VertxPluginRegistry",
-      "config": {}
+      "config": {
+        //"pluginRepositories":["https://url.to.maven.repo"],
```

Comment:
Does this work?  Comments in JSON?

Reply:
Normally, in pure JSON no. But in Vert.X, parser seems to be able to handle C-style comments.

It is described in Vert.X documentation : https://apiman.gitbooks.io/apiman-installation-guide/installation-guide/vertx/config-guide.html

Reply:
Cool!